packages/react/src/ButtonGroup/ButtonGroup.module.css

```diff
@@ -0,0 +1,8 @@
+.ButtonGroup {
+  display: inline-grid;
+  gap: 8px;
+}
+
+.ButtonGroup--horizontal {
+  grid-template-columns: auto auto;
+}
```

Comment:
Are you missing styles for vertical? I can see it listed in the direction props.

Reply:
vertical styles happen implicitly I think relying on browsers for this would make more sense. We can update this if the responsive update you mentioned affects this.